alacritty/src/event.rs

```diff
@@ -629,8 +629,9 @@ impl<N: Notify + OnResize> Processor<N> {
                         }
                     },
                     WindowEvent::DroppedFile(path) => {
+                        const PATH_SEPARATOR: &str = " ";
```

Comment:
No point in defining this as a constant here locally. We can just inline it.

Reply:
I've removed the constant definition and just used the string literal.